### DISCUSSION TITLE
a/hikvision-nvr-playback-with-gb28181

### DISCUSSION THREAD

**Discussion by giscus** [sincerefly/giscus-data#3] [Announcements]
# a/hikvision-nvr-playback-with-gb28181

https://blog.yasking.org/a/hikvision-nvr-playback-with-gb28181.html

**Comment by Johnny-Yh** [comment]
感谢分享！

**Reply by sincerefly** [reply]
对大家有帮助就好 🌹

**Comment by luhengyu1** [comment]
请问这些信令是根据国标协议来的吗，我尝试了录像回放的倍速功能，nvr一直返回400，不知道怎么排查

**Reply by sincerefly** [reply]
是的，基于国标进行的测试，不同厂商、型号 NVR 和软件版本可能会有差异，没什么好办法，多测试验证，或者找厂家工程师协助看日志排查。
